app/src/main/java/com/quran/labs/androidquran/ui/fragment/AyahTranslationFragment.java

```diff
@@ -149,7 +151,18 @@ public void refreshView() {
       final int verses = 1 + Math.abs(
           quranInfo.getAyahId(start.sura, start.ayah) - quranInfo.getAyahId(end.sura, end.ayah));
       VerseRange verseRange = new VerseRange(start.sura, start.ayah, end.sura, end.ayah, verses);
-      translationPresenter.refresh(verseRange);
+
+      List<String> translationsFileNames = new ArrayList<>(quranSettings.getActiveTranslations());
+      List<String> orderedTranslationsFilesNames = new ArrayList<>();
+
+      for (LocalTranslation translation : allTranslationsFromDb) {
+        for (int i = 0; i < translationsFileNames.size(); i++) {
+          if (translation.getFilename().equals(translationsFileNames.get(i))) {
+            orderedTranslationsFilesNames.add(translationsFileNames.get(i));
+          }
+        }
+      }
+      translationPresenter.refresh(new ArrayList<>(orderedTranslationsFilesNames), verseRange);
     }
```

Comment:
let's remove this as per my comment above

Reply:
ok